### ISSUE TITLE
[ACCESSIBILITY]: check color contrast of all elements with both themes

### ISSUE THREAD

**Issue by michaella23** [michaella23/bootcamp-collab#120]
This should be an issue that we are taking into consideration with every iteration of the project. 

According to WCAG 2.1 Guidelines, the minimum contrast elements on the page should have to be accessible to all users will have an AA rating.

>>> The visual presentation of [text](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html#dfn-text) and [images of text](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html#dfn-image-of-text) has a [contrast ratio](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html#dfn-contrast-ratio) of at least 4.5:1, except for the following:

Large Text
[Large-scale](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html#dfn-large-scale) text and images of large-scale text have a contrast ratio of at least 3:1;

Incidental
Text or images of text that are part of an inactive [user interface component](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html#dfn-user-interface-component), that are [pure decoration](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html#dfn-pure-decoration), that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.

Logotypes
Text that is part of a logo or brand name has no contrast requirement.
